pandas/io/_util.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+import pandas as pd
+
+
+def _arrow_dtype_mapping(api) -> dict:
```

Comment:
Might be just worth calling `import_option_dependency` here since `api` always need to be a pyarrow module?

Reply:
Yeah makes sense, changed